ts/components/ui/AppHeader.tsx

```diff
@@ -1,10 +1,26 @@
 import { Header, NativeBase } from "native-base";
 import * as React from "react";
+import { StyleSheet } from "react-native";
 import variables from "../../theme/variables";
 import ConnectionBar from "../ConnectionBar";
 
 type Props = NativeBase.Header;
 
+const styles = StyleSheet.create({
+  header: {
```

Comment:
Can't we set the style to the header theme directly?

Reply:
fixed